cmd/soroban-rpc/internal/methods/get_version_info.go

```diff
@@ -0,0 +1,30 @@
+package methods
+
+import (
+	"context"
+	"github.com/creachadair/jrpc2"
+	"github.com/creachadair/jrpc2/handler"
+	"github.com/stellar/soroban-rpc/cmd/soroban-rpc/internal/config"
+)
+
+type GetVersionInfoRequest struct {
+}
+
+type GetVersionInfoResponse struct {
+	Version                string `json:"version"`
+	CommitHash             string `json:"commit_hash"`
+	BuildTimestamp         string `json:"build_time_stamp"`
+	CaptiveCoreVersionInfo string `json:"captive_core_version"`
+}
+
+func NewGetVersionInfoHandler() jrpc2.Handler {
+	return handler.New(func(ctx context.Context, request GetVersionInfoRequest) (GetVersionInfoResponse, error) {
+
+		return GetVersionInfoResponse{
+			Version:                config.Version,
+			CommitHash:             config.CommitHash,
+			BuildTimestamp:         config.BuildTimestamp,
+			CaptiveCoreVersionInfo: config.CaptiveCoreVersionInfo,
```

Comment:
this assumes that the captive core version is static but sometimes we rollout new versions of core while horizon / soroban-rpc is still running:

https://github.com/stellar/go/issues/3602

Reply:
You may want to reuse https://github.com/stellar/soroban-rpc/pull/142

Reply:
Thank you, Used `/info` endpoint to fetch captive core version and reused `getBucketListSizeAndProtocolVersion` to fetch protocol version